doc/INSTALL

```diff
@@ -176,9 +176,9 @@ that probably work fine but these instructions are for "Homebrew":
   5. Configure, possibly adding a CC option for pointing to a specific gcc and
      using whatever LDFLAGS and CPPFLAGS was recommended when you installed
      Homebrew's OpenSSL:
-        ./configure CC="/opt/homebrew/bin/gcc-10" \
-          LDFLAGS="-L/opt/homebrew/opt/openssl/lib" \
-          CPPFLAGS="-I/opt/homebrew/opt/openssl/include"
+        ./configure CC="/usr/local/bin/gcc-13" \
+          LDFLAGS="-L/usr/local/opt/openssl/lib " \
+          CPPFLAGS="-I/usr/local/opt/openssl/include"
```

Comment:
I think I recall noticing that my MacBook has it one of the ways (i.e. the `/opt` vs. `/usr/local`) while the M1 I can borrow sporadically has the other. Both have Homebrew installed more or less the same way. Either it's because mine had HomeBrew for two years longer, or it's somehow caused by the fact the latter is M1 while I still have an intel.